addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -778,3 +778,8 @@ msgstr ""
 msgctxt "#31167"
 msgid "Animate background"
 msgstr ""
+
+#: /xml/DialogVideoInfo.xml
```

Comment:
Mind adding a little contextual note `#. description of how this label is used` to help translators find the correct translation? Thanks